flood_adapt/integrator/sfincs_adapter.py

```diff
@@ -291,7 +291,10 @@ def add_floodwall(self, floodwall: FloodWallModel, measure_path=Path):
 
         # Add floodwall attributes to geodataframe
         gdf_floodwall["name"] = floodwall.name
-        gdf_floodwall["z"] = floodwall.elevation.convert(UnitTypesLength("meters"))
+        if (gdf_floodwall.geometry.type == "MultiLineString").any():
+            gdf_floodwall = gdf_floodwall.explode()
+        if "z" not in gdf_floodwall.columns:
+            gdf_floodwall["z"] = floodwall.elevation.convert(UnitTypesLength("meters"))
         gdf_floodwall["par1"] = 0.6
```

Comment:
Even thought this is not a new addition I am curious on what this par1 is :)

Reply:
I think we spoke about it